### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

**Comment by sbose78** [comment]
Hi! Could you tell me who's going to consume this?

**Comment by ldimaggi** [comment]
OpenShift CI jobs are failing without this Dockerfile:
ci/rehearse/redhat-developer/helm/master/images




2019/11/11 17:58:13 Resolved source https://github.com/openshift/release to
master@8dff90f0, merging: #5685 a7bad30d @ldimaggi
2
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A2>
2019/11/11 17:58:13 Resolved source https://github.com/redhat-developer/helm
to master@
3
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A3>
2019/11/11 17:58:13 Resolved openshift/release:golang-1.12 to
sha256:912118829bf9fcc2fb00d3ee4900f7861618f1ccfccb0b43de3eb1c855aa3b29
4
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A4>
2019/11/11 17:58:13 Resolved ocp/4.2:base to
sha256:943b2891a7255a347f51b28a372ecfe438eb5522dad43b652fb2e490ffac7f95
5
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A5>
2019/11/11 17:58:13 Using namespace ci-op-grsl577c
6
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A6>
2019/11/11 17:58:13 Running [input:root], [input:base], [release-inputs],
src, helm, [output:stable:helm], [images]
7
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A7>
2019/11/11 17:58:13 Creating namespace ci-op-grsl577c
8
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A8>
2019/11/11 17:58:13 Creating rolebinding for user ldimaggi in namespace
ci-op-grsl577c
9
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A9>
2019/11/11 17:58:13 Setting a soft TTL of 1h0m0s for the namespace
10
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A10>
2019/11/11 17:58:13 Setting a hard TTL of 12h0m0s for the namespace
11
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A11>
2019/11/11 17:58:13 Setting up pipeline imagestream for the test
12
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A12>
2019/11/11 17:58:13 Tagging openshift/release:golang-1.12 into pipeline:root
13
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A13>
2019/11/11 17:58:13 Tagging ocp/4.2:base into pipeline:base
14
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A14>
2019/11/11 17:58:13 Tagged shared images from ocp/4.2:${component}, images
will be pullable from
registry.svc.ci.openshift.org/ci-op-grsl577c/stable:${component}
15
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A15>
2019/11/11 17:58:13 Building src
16
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A16>
2019/11/11 18:00:57 Build src succeeded after 2m55s
17
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A17>
2019/11/11 18:00:57 Building helm
18
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A18>
2019/11/11 18:01:11 Build helm failed, printing logs:
19
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A19>
Pulling image
"docker-registry.default.svc:5000/ci-op-grsl577c/pipeline@sha256:f7cc5e1705b66642c26773816ca3811a835866c255f0307fc32b8d4d202581cf"
...
20
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A20>
error: open /tmp/build/inputs/Dockerfile: no such file or directory
21
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A21>
2019/11/11 18:01:22 Ran for 3m9s
22
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A22>
2019/11/11 18:01:23 Submitted failure event to sentry
(id=ab2890cb37914009b9d2bdf24e31ac69)
23
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A23>
error: could not run steps: step helm failed: could not wait for build: the
build helm failed after 13s with reason GenericBuildFailed: Generic Build
failure - check logs for details.
24
<https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5685/rehearse-5685-pull-ci-redhat-developer-helm-master-images/17#1:build-log.txt%3A24>

**Comment by ldimaggi** [comment]
OpenShift CI jobs are failing without this Dockerfile: 
ci/rehearse/redhat-developer/helm/master/images

```
2019/11/11 17:39:41 Resolved source https://github.com/openshift/release to master@8dff90f0, merging: #5685 75ff19c2 @ldimaggi
2019/11/11 17:39:41 Resolved source https://github.com/redhat-developer/helm to master@
2019/11/11 17:39:41 Resolved openshift/release:golang-1.12 to sha256:912118829bf9fcc2fb00d3ee4900f7861618f1ccfccb0b43de3eb1c855aa3b29
2019/11/11 17:39:41 Resolved ocp/4.2:base to sha256:943b2891a7255a347f51b28a372ecfe438eb5522dad43b652fb2e490ffac7f95
2019/11/11 17:39:41 Using namespace ci-op-rw358kl4
2019/11/11 17:39:41 Running [input:root], [input:base], [release-inputs], src, helm, [output:stable:helm], [images]
2019/11/11 17:39:41 Creating namespace ci-op-rw358kl4
2019/11/11 17:39:41 Creating rolebinding for user ldimaggi in namespace ci-op-rw358kl4
2019/11/11 17:39:41 Setting a soft TTL of 1h0m0s for the namespace
2019/11/11 17:39:41 Setting a hard TTL of 12h0m0s for the namespace
2019/11/11 17:39:41 Setting up pipeline imagestream for the test
2019/11/11 17:39:41 Tagging openshift/release:golang-1.12 into pipeline:root
2019/11/11 17:39:41 Tagging ocp/4.2:base into pipeline:base
2019/11/11 17:39:41 Tagged shared images from ocp/4.2:${component}, images will be pullable from registry.svc.ci.openshift.org/ci-op-rw358kl4/stable:${component}
2019/11/11 17:39:41 Building src
2019/11/11 17:40:21 Build src succeeded after 42s
2019/11/11 17:40:21 Building helm
2019/11/11 17:40:30 Build helm failed, printing logs:
Pulling image "docker-registry.default.svc:5000/ci-op-rw358kl4/pipeline@sha256:2c5e7978fdb04d343b1ccbd5317c5c769904674b70797108eeec355d618504aa" ...
error: open /tmp/build/inputs/Dockerfile: no such file or directory
2019/11/11 17:40:41 Ran for 1m0s
2019/11/11 17:40:42 Submitted failure event to sentry (id=6478fd75fe324a409dfc6455878b2ae0)
error: could not run steps: step helm failed: could not wait for build: the build helm failed after 8s with reason GenericBuildFailed: Generic Build failure - check logs for details.
```

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ENV GIT_COMMITTER_EMAIL devtools@redhat.com
+ENV PATH=:$GOPATH/bin:/tmp/goroot/go/bin:$PATH
+
+WORKDIR /tmp
+
+RUN mkdir -p $GOPATH/bin
+RUN mkdir -p /tmp/goroot
+
+RUN curl -Lo go1.12.6.linux-amd64.tar.gz https://dl.google.com/go/go1.12.6.linux-amd64.tar.gz && tar -C /tmp/goroot -xzf go1.12.6.linux-amd64.tar.gz
+
```
Comment (bamachrn) [inline Dockerfile:16]: 1. why don't we use golang:1.12 as base image? I believe we can get the go variables set there.
2. I thought we are going to use this image to run builds and tests on top of it. do we not need the source code inside it? I mean, where are we downloading the source to build?

**Comment by ldimaggi** [comment]
Closing this pull request in favor of creating a new pull request.

**Pull request closed without merging**